crowbar_framework/app/models/ceph_service.rb

```diff
@@ -289,6 +289,21 @@ def validate_proposal_after_save proposal
       end
     end
 
+    nodes_without_suitable_drives = proposal["deployment"][@bc_name]["elements"]["ceph-osd"].select do |node_name|
+      node = NodeObject.find_node_by_name(node_name)
+      if node.nil?
+          false
+      else
+          disks_count = node.unclaimed_physical_drives.length 
+          disks_count += node.physical_drives.select { |d, data| node.disk_owner(node.unique_device_for(d)) == 'Ceph' }.length
+          disks_count == 0
+      end
+    end
+    unless nodes_without_suitable_drives.empty?
```

Comment:
Add an empty line here and remove one of the double new line after this block :)